livedata/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -380,7 +381,7 @@ class ChatDomainImpl private constructor(
 
     fun channel(cid: String): ChannelControllerImpl {
         val parts = cid.split(":")
-        check(parts.size == 2) { "Received invalid cid, expected format messaging:123, got $cid" }
+        check(parts.size >= 2) { "Received invalid cid, expected format messaging:123, got $cid" }
```

Comment:
Why do we accept more than 2 parts on the `cid`? 
Is there any case where `cid` is something different than `xxx:xxx`

Reply:
Channel name regex doesn't allow ":" in channel name now. So, I'm switching back to "==".